app/graphql/types/library_entry_status.rb

```diff
@@ -0,0 +1,7 @@
+class Types::LibraryEntryStatus < Types::BaseEnum
+  value 'current', 'Media you currently read/watch.'
+  value 'planned', 'Media you plan to read/watch at some point.'
+  value 'completed', 'Media you have completed.'
+  value 'on_hold', 'Media you have started but decided to stop.'
+  value 'dropped', 'Media you will not complete.'
```

Comment:
We should pick either second or third person in docs and stick to it.

So like, in third person:
```suggestion
  value 'current', 'The user is currently reading or watching this media'
  value 'planned', 'The user plans to read or watch this media in future'
  value 'completed', 'The user completed this media'
  value 'on_hold', 'The user started but paused reading or watching this media'
  value 'dropped', 'The user started but chose not to finish this media'
```

Reply:
Ok, Also just realized I should upcase all values `CURRENT` just to be consistent for how other enums are.